spec/controllers/users_controller_spec.rb

```diff
@@ -5,33 +5,23 @@
   let(:user_response) { JSON.parse(response.body, symbolize_names: true) }
 
   describe "GET #show" do
-    context "when user is signed in" do
-      before :each do
-        sign_in user
-      end
+    before :each do
```

Comment:
this is actually a great use for context, to separate escenarios like you had done. Next time leave the the context